aclk/mqtt.c

```diff
@@ -113,10 +79,20 @@ int lws_wss_client_initialized = 0;
 int mqtt_over_websockets = 1;
```

Comment:
It might prove usefull when we have MQTT over HTTPS etc. etc.

Reply:
Will handle in the next issue

Reply:
Hi if so we should turn it into enum MQTT_TRANSPORT_TYPE to support later MQTT over HTTPS fallback?

Reply:
The HTTPS fall-back option has not been designed or groomed yet. We do not know how the transport layers in it will function. Remove the parts of the code that do not match our use-cases. When we know what the new use-cases are then we will work out what to add.